Enabling unknown collectors leads to crash

I run wmi_exporter 0.11.1 (the 64 bit variant) on my Windows 10 x64 1909 using this command line:

```wmi_exporter.exe --collectors.enabled "cpu,cs,logical_disk,logon,memory,net,os,process,remote_fx,system,tcp,terminal_services" --collector.process.whitelist="euca.+"```

Unfortunately it fails with this:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x932956]

goroutine 1 [running]:
github.com/martinlindhe/wmi_exporter/collector.Build(...)
        C:/gopath/src/github.com/martinlindhe/wmi_exporter/collector/collector.go:75
main.loadCollectors(0xc000054060, 0x56, 0xc00009e000, 0xc000158000, 0xc00005e178)
        C:/gopath/src/github.com/martinlindhe/wmi_exporter/exporter.go:235 +0xc6
main.main()
        C:/gopath/src/github.com/martinlindhe/wmi_exporter/exporter.go:322 +0xeeb
???

--- a/collector/collector.go
+++ b/collector/collector.go
@@ -1,6 +1,7 @@
 package collector
 
 import (
+	"fmt"
 	"strconv"
 	"strings"
 
@@ -72,7 +73,11 @@ func Available() []string {
 	return cs
 }
 func Build(collector string) (Collector, error) {
-	return builders[collector]()
+	builder, exists := builders[collector]
+	if !exists {
+		return nil, fmt.Errorf("Unknown collector %q", collector)
+	}
+	return builder()
 }
 func getPerfQuery(collectors []string) string {
 	parts := make([]string, 0, len(collectors))

